ogusa/parameters.py

```diff
@@ -396,16 +396,16 @@ def get_parameters(test=False, baseline=False, guid='', user_modifiable=False, m
     mtry_params = dict_params['tfunc_mtry_params_S'][:S,:BW,:]
 
     # # Make all ETRs equal the average
-    etr_params = np.zeros(etr_params.shape)
-    etr_params[:, :, 10] = dict_params['tfunc_avg_etr'] # set shift to average rate
+    # etr_params = np.zeros(etr_params.shape)
+    # etr_params[:, :, 10] = dict_params['tfunc_avg_etr'] # set shift to average rate
 
     # # Make all MTRx equal the average
-    mtrx_params = np.zeros(mtrx_params.shape)
-    mtrx_params[:, :, 10] = dict_params['tfunc_avg_mtrx'] # set shift to average rate
+    # mtrx_params = np.zeros(mtrx_params.shape)
+    # mtrx_params[:, :, 10] = dict_params['tfunc_avg_mtrx'] # set shift to average rate
 
     # # Make all MTRy equal the average
-    mtry_params = np.zeros(mtry_params.shape)
-    mtry_params[:, :, 10] = dict_params['tfunc_avg_mtry'] # set shift to average rate
+    # mtry_params = np.zeros(mtry_params.shape)
+    # mtry_params[:, :, 10] = dict_params['tfunc_avg_mtry'] # set shift to average rate
```

Comment:
@jdebacker why did you comment these lines out?

Reply:
@hdoupe I was working with the progressive rate functions.  But I didn't want to include that change in this PR - thanks for catching it!  Updated with a new commit.